streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1164,9 +1243,12 @@ public void shouldHandleMultipleRemovedTasksFromStateUpdater() {
         final StreamTask taskToUpdateInputPartitions = statefulTask(taskId03, taskId03ChangelogPartitions)
             .inState(State.RESTORING)
             .withInputPartitions(taskId03Partitions).build();
+        final StreamTask taskToCloseReviveAndUpdateInputPartitions = statefulTask(taskId04, taskId04ChangelogPartitions)
+                .inState(State.RESTORING)
+                .withInputPartitions(taskId04Partitions).build();
```

Comment:
nit:
```suggestion
            .inState(State.RESTORING)
            .withInputPartitions(taskId04Partitions).build();
```

Reply:
Done

Reply:
This does not seem done. But nevermind, if it is just this I will of course not block the PR.

Reply:
I think there was a similar indentation issue on line 1713 that you fixed. I also saw one on line 1652.

Reply:
Right, I only fixed the other one. I think I recently changed my IDE to use 8 character continuation indentation, became somebody complained about it in a client PR. Changing it back. Honestly, I wish we'd just have automatic formatting in kafka.